src/OSPSuite.Core/Domain/Builder/PathAndValueEntity.cs

```diff
@@ -62,23 +61,13 @@ public Unit DisplayUnit
          set => SetProperty(ref _displayUnit, value);
       }
 
-      public ValueOrigin ValueOrigin { get; }
 
       public IObjectPath Path
       {
          get => ContainerPath.Clone<IObjectPath>().AndAdd(Name);
          set => entityFullPathToComponents(value);
       }
 
-      public void UpdateValueOriginFrom(ValueOrigin sourceValueOrigin)
```

Comment:
Why don't they have value origin?
 Why was this removed?

Reply:
We didn't implement the binder in the view for ExpressionProfile so I didn't think it was necessary. If all PathAndValueEntity should have it then let me know, but then it probably makes sense to expose it in the view.